### PR TITLE
Find: Highlight selected nodes

### PR DESCRIPTION
**Pull request by vperelmax**
Initial Implementation
Find on graph selected via Gremlin Expression sub-graph and highlight it.

### CODE REVIEW

File: statics/js/components/topology.js [hunk 1]
```diff
@@ -309,6 +332,31 @@ var TopologyComponent = {
...
+    highlightSelectedNodes: function(bool) {
+      var self = this;
+
+      this.$topologyQuery(this.topologySelect)
+        .then(function(nodes) {
+          nodes.forEach(function(n) {
+            for (var i in n.Nodes) {
+                var myNode = n.Nodes[i];
+                if (bool) {
+                    self.$store.commit("highlight", myNode.ID);
```
Comment (safchain) [inline statics/js/components/topology.js]: you should use the self.layout.highlightNodeID function as it is the layout which has to handle this.

File: statics/js/components/topology.js [hunk 2]
```diff
@@ -122,6 +135,7 @@ var TopologyComponent = {
       timeRange: [-120, 0],
       collapsed: false,
       topologyFilter: "",
+      topologySelect: "",
```
Comment (safchain) [inline statics/js/components/topology.js]: nit: I would not use "topologySelect" as it lead to think that we select something. Here it is just matter of highlighting

File: statics/js/components/topology.js (hunk 1 above)
Comment (vperelmax) [reply statics/js/components/topology.js]: sorry for a stupid question - do you mean without $store? I thought I want to do it asynchronously. Actually copied the code from capturing.

Comment (safchain) [reply statics/js/components/topology.js]: There is a dedicated function here: 
https://github.com/skydive-project/skydive/blob/master/statics/js/components/topology.js#L164

I think we need to change the capture stuff but that another story. So if you wanted to stay with your implementation that is OK for me I will change it in my "sliding panel" patch.

**Comment by vperelmax** [comment]
I have issue with cleaning old selection before running a new one - I will fix it and upload once again.

File: statics/js/components/topology.js (hunk 1 above)
Comment (vperelmax) [reply statics/js/components/topology.js]: I will change it

Comment (vperelmax) [reply statics/js/components/topology.js]: It looks like it has bug.. I will check it - the subscribe function sends to highlightNodeID one parameter, while highlightNodeID in layout receives two parameters and doesnt actually use the second one...
Maybe my misunderstanding of JS :(

**Comment by vperelmax** [comment]
1) The issue was resolved and I have fixed also everything to "highlight"
2) Now user should not use the suffix "SubGraph()" in both filter and highlight requests and the selection is done on the flight (changing state of the graph every time the user types ")")

**Pull request merged**